vendor/github.com/containerd/containerd/pull.go

```diff
@@ -32,8 +33,16 @@ import (
 
 // Pull downloads the provided content into containerd's content store
 // and returns a platform specific image object
-func (c *Client) Pull(ctx context.Context, ref string, opts ...RemoteOpt) (_ Image, retErr error) {
+func (c *Client) Pull(ctx context.Context, ref string, timeout time.Duration, opts ...RemoteOpt) (_ Image, retErr error) {
```

Comment:
To update vendor files we modify vendor.conf at the root of this repo in your branch.. then run the tool "vndr".  vndr will bring in the necessary vendor/ files then you add a commit for your change to vendor.conf and vendor/

Reply:
OK, thx!